lib/telesign/helpers.rb

```diff
@@ -11,6 +11,8 @@ def validate_response response
       if response.status != 200
```

Comment:
perhaps you can do 
    if response.status != 200 && response.status != 400 
and avoid adding the 2 new lines. The things inside the if block are throwing exceptions and the response is being returned outside, so it is better to not mix the 2.